src/blocks/venue/edit.js

```diff
@@ -212,6 +223,45 @@ const Edit = ({ attributes, setAttributes, isSelected }) => {
 							min={100}
 							max={1000}
 						/>
+						<PanelRow>
+							{__('Latitude / Longitude', 'gatherpress')}
+						</PanelRow>
+						<PanelRow>
+							<ToggleControl
+								label={
+									mapCustomLatLng
+										? __('Use custom values', 'gatherpress')
+										: __(
+												'Use default values',
+												'gatherpress'
+											)
+								}
+								checked={mapCustomLatLng}
+								onChange={(value) => {
+									setAttributes({ mapCustomLatLng: value });
+								}}
+							/>
+						</PanelRow>
+						{mapCustomLatLng && (
+							<>
+								<TextControl
+									label={__('Latitude', 'gatherpress')}
+									value={latitude}
+									onChange={(value) => {
+										Broadcaster({ setLatitude: value });
+										updateVenueMeta({ latitude: value });
+									}}
+								/>
+								<TextControl
```

Comment:
`NumberControl` here too. If you leave this as `TextControl`, hitting a non-number crashes the block.